desktop.css

```diff
@@ -549,6 +605,48 @@
     height: 40px;
   }
 
+  #rect {
+    position: relative;
+    left: 30%;
+    top: 12.5%;
+  }
+
+  #rect2 {
+    position: relative;
+    top: 14%;
+    left: 27%;
+  }
+
+  #rect3 {
+    position: relative;
+    top: 14%;
+    left: 40%;
+  }
+
+  #l1 {
+    position: relative;
+    bottom: 10px;
+  }
+
+  .desktop-posts-section {
```

Comment:
As highlighted by the previous reviewer, please fix the margins on the posts. Your margins are too big.
## Your design
<img width="1675" alt="Screenshot 2022-10-01 at 18 46 06" src="https://user-images.githubusercontent.com/48546992/193417518-7ce95285-0765-4f90-88e7-adc2547e656c.png">

##Figma template
<img width="715" alt="Screenshot 2022-10-01 at 18 46 45" src="https://user-images.githubusercontent.com/48546992/193417555-1da4d901-a10c-4e49-beb6-d891087b6d9c.png">